### PR TITLE
Improve Services documentation

### PR DESCRIPTION
**Pull request by CameronMunroe**
I updated the services page for a few of the following reasons:

* Fix placement of certain info that was better placed elsewhere.
* Adjust the documentation to inculde how to install during setup step.
* Fixed a few lines for a better feel to the wording.

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Extensions/Services.md [hunk 1]
```diff
@@ -4,39 +4,41 @@ path: blob/master/doc/
...
+The `check_` prefix is stripped out when displaying in the "Add Service"
+GUI "Type" dropdown list.
 
 ## Setup
 
-Service checks is now distributed aware. If you run a distributed
-setup then you can now run `services-wrapper.py` in cron instead of
-`check-services.php` across all polling nodes.
+Service checks are now distributable if you run a distributed
+setup. To leverage this, use `services-wrapper.py` in cron instead of
```
Comment (murrant) [inline doc/Extensions/Services.md:29]: The dispatch service also works with services, not just the wrapper.

Comment (CameronMunroe) [reply doc/Extensions/Services.md]: Please let me know if the new bit about the `dispatcher` service is functional for this.

Comment (murrant) [reply doc/Extensions/Services.md]: yes :)

**Review by murrant — APPROVED** [review]
Thanks :)

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-69-release-changelog-october-2020/13837/1

**Pull request merged**